packages/specification/schemas/resources/MedicationRequest.yaml

```diff
@@ -19,25 +17,13 @@ properties:
     type: string
     description: FHIR resource type.
     enum: [MedicationRequest]
-  extension:
-    type: array
-    description: FHIR extension array.
-    items:
-      oneOf:
-        - $ref: ../extensions/DispensingInformation.yaml
-        - $ref: ../extensions/MedicationRepeatInformation.yaml
-  identifier:
-    type: array
-    description: The unique identifier for this line item on the prescription.
-    items:
-      $ref: ../fragments/identifiers.yaml#/PrescriptionOrderItemNumber
   status:
     type: string
-    description: The status of a MedicationRequest object.
-    enum: [active, cancelled, completed]
+    description: The status of the line item within the prescription. Active is used while ever the 
```

Comment:
Incomplete sentence. 

Maybe express this over a series of lines:

`active` means the item is ready to be dispensed or currently in the process of being dispensed.
`cancelled` means the item has been cancelled by the prescriber.
`stopped` means the item has not been issued to the patient by the dispenser.